tests/integration/targets/mso_tenant/tasks/main.yml

```diff
@@ -243,7 +278,7 @@
 - name: nm_add_tenant3_with_invalid_user
   assert:
     that:
-    - nm_add_tenant3_with_invalid_user is not changed
+    - nm_add_tenant3_with_invalid_user is changed
```

Comment:
The assertion check failed! Please check it again.

Reply:
fixed.